src/Symfony/Component/HttpKernel/Tests/Bundle/BundleTest.php

```diff
@@ -32,24 +29,6 @@ public function testGetContainerExtension()
     }
 
     /**
-     * @group legacy
-     * @expectedDeprecation Auto-registration of the command "Symfony\Component\HttpKernel\Tests\Fixtures\ExtensionPresentBundle\Command\FooCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead.
-     */
-    public function testRegisterCommands()
```

Comment:
I would keep this test and assert no command is registered for `ExtensionPresentBundle`.